IapExample/react-native.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    dependencies: {
+      'react-native-iap': {
+        platforms: {
+          ios: null, // disable ios platform, so we can use local filess
+        },
+      },
+    },
+  };
```

Comment:
I'll just point out the reference for help.
https://github.com/react-native-community/cli/blob/main/docs/dependencies.md

Also the file looks like it has an extra tab. But we can handle that in the other PR.